sympy/physics/mechanics/tests/test_actuator.py

```diff
@@ -849,3 +852,144 @@ def test_to_loads(self):
+
+        # Mass, gravity constant, friction coefficient, coefficient of Stribeck friction, viscous_coefficient
+        self.m, self.g, self.mu_k, self.mu_s, self.v_s, self.sigma, self.F = symbols('m g mu_k mu_s v_s sigma F', real=True)
+
+        # Define the reference frame
+        self.N = ReferenceFrame('N')
+        self.O = Point('O')
+        self.P = self.O.locatenew('P', self.q1 * self.N.x)
+        self.O.set_vel(self.N, 0)
+        self.P.set_vel(self.N, self.u1 * self.N.x)
+        self.pathway = LinearPathway(self.O, self.P)
+
+        self.particle = Particle('particle', self.P, self.m) # Block
+
+        self.system = System(self.N, self.O)
+        self.system.q_ind = [self.q1]
+        self.system.u_ind = [self.u1]
+        self.system.kdes = [self.q1.diff() - self.u1]
+        self.system.add_bodies(self.particle)
+        self.system.apply_uniform_gravity(-self.g * self.N.y)
+        self.system.add_loads(Force(self.particle, self.F * self.N.x))
+
+    def test_block_on_surface_default(self):
+        friction = CoulombKineticFriction(
+            self.mu_k,
+            self.m * self.g,
+            self.pathway,
+        )
+        self.system.add_actuators(friction)
+
+        # Positive velocity case
+        eoms = self.system.form_eoms()
+        expected_positive = self.F - self.g * self.m * self.mu_k - self.m * self.u1.diff()
+        assert (eoms[0] - expected_positive).subs(self.repl_positive) == 0
+
+        # Negative velocity case
+        eoms = self.system.form_eoms()
+        expected_negative = self.F + self.g * self.m * self.mu_k - self.m * self.u1.diff()
+        assert (eoms[0] - expected_negative).subs(self.repl_negative) == 0
+
+    def test_block_on_surface_viscous(self):
+        friction = CoulombKineticFriction(
+            self.mu_k,
+            self.m * self.g,
+            self.pathway,
+            sigma=self.sigma
+        )
+        self.system.add_actuators(friction)
+
+        # Positive velocity case
+        eoms = self.system.form_eoms()
+        expected_positive = self.F - self.g * self.m * self.mu_k + self.sigma * self.u1 - self.m * self.u1.diff()
+        assert (eoms[0] - expected_positive).subs(self.repl_positive) == 0
+
+        # Negative velocity case
+        eoms = self.system.form_eoms()
+        expected_negative = self.F + self.g * self.m * self.mu_k + self.sigma * self.u1 - self.m * self.u1.diff()
+        assert (eoms[0] - expected_negative).subs(self.repl_negative) == 0
+
+    def test_block_on_surface_stribeck(self):
+        friction = CoulombKineticFriction(
+            self.mu_k,
+            self.m * self.g,
+            self.pathway,
+            v_s=self.v_s,
+            mu_s=self.mu_k
+        )
+        self.system.add_actuators(friction)
+
+        # Positive velocity case
+        eoms = self.system.form_eoms()
+        expected_positive = self.F - self.g * self.m * self.mu_k - self.m * self.u1.diff()
```

Comment:
Where is the Stribeck term in this equation?

Reply:
The Stribeck term is eliminated when we set mu_s = mu_k and doesn't appear in the eom, but when we set mu_s = mu_s or something else, the eom includes the Stribeck term, i.e., `(-g*m*mu_k + g*m*mu_s)*exp(-sign(q1(t))**2*Derivative(q1(t), t)**2/v_s**2)`

Reply:
This test checks the Stribeck term, so it is best if mu_s is a separate symbol, such that the term isn't eliminated.

Reply:
I updated `test_block_on_surface_stribeck` and `test_block_on_surface_all` to include the Stribeck term.